<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>700</width>
                <height>600</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>700</width>
                <height>600</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>700</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>NovelEditTools</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTabWidget" name="tabWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>681</width>
                        <height>201</height>
                    </rect>
                </property>
                <property name="currentIndex">
                    <number>1</number>
                </property>
                <widget class="QWidget" name="tab">
                    <attribute name="title">
                        <string>网页下载</string>
                    </attribute>
                    <widget class="QLineEdit" name="save_novel_path_by_page">
                        <property name="geometry">
                            <rect>
                                <x>281</x>
                                <y>95</y>
                                <width>211</width>
                                <height>25</height>
                            </rect>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="pushButton">
                        <property name="geometry">
                            <rect>
                                <x>200</x>
                                <y>95</y>
                                <width>75</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>11</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>保存目录</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="button_download_start_executr">
                        <property name="geometry">
                            <rect>
                                <x>290</x>
                                <y>130</y>
                                <width>100</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>开始下载</string>
                        </property>
                    </widget>
                    <widget class="QComboBox" name="select_download_type">
                        <property name="geometry">
                            <rect>
                                <x>280</x>
                                <y>20</y>
                                <width>111</width>
                                <height>28</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>12</pointsize>
                            </font>
                        </property>
                        <item>
                            <property name="text">
                                <string>shubao12.com</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>tmallyh.top</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>aastory.space</string>
                            </property>
                        </item>
                    </widget>
                    <widget class="QLabel" name="label_2">
                        <property name="geometry">
                            <rect>
                                <x>200</x>
                                <y>20</y>
                                <width>71</width>
                                <height>28</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>12</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>下载网站</string>
                        </property>
                    </widget>
                    <widget class="QComboBox" name="select_download_mode">
                        <property name="geometry">
                            <rect>
                                <x>400</x>
                                <y>20</y>
                                <width>91</width>
                                <height>28</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>12</pointsize>
                            </font>
                        </property>
                        <item>
                            <property name="text">
                                <string>批量下载</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>单章下载</string>
                            </property>
                        </item>
                    </widget>
                    <widget class="QLabel" name="label_3">
                        <property name="geometry">
                            <rect>
                                <x>200</x>
                                <y>60</y>
                                <width>71</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>12</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>下载地址</string>
                        </property>
                    </widget>
                    <widget class="QLineEdit" name="input_download_url">
                        <property name="geometry">
                            <rect>
                                <x>280</x>
                                <y>60</y>
                                <width>211</width>
                                <height>25</height>
                            </rect>
                        </property>
                    </widget>
                </widget>
                <widget class="QWidget" name="tab_3">
                    <attribute name="title">
                        <string>文本处理</string>
                    </attribute>
                    <widget class="QPushButton" name="manual_button_get_file_list">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>10</y>
                                <width>71</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>打开文件</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_other_save_file">
                        <property name="geometry">
                            <rect>
                                <x>268</x>
                                <y>10</y>
                                <width>71</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>另存为</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_save_file">
                        <property name="geometry">
                            <rect>
                                <x>178</x>
                                <y>10</y>
                                <width>71</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>保存文件</string>
                        </property>
                    </widget>
                    <widget class="QListWidget" name="manual_file_item_list">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>50</y>
                                <width>251</width>
                                <height>105</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <family>Arial</family>
                                <pointsize>13</pointsize>
                            </font>
                        </property>
                    </widget>
                    <widget class="QLabel" name="label_5">
                        <property name="geometry">
                            <rect>
                                <x>365</x>
                                <y>11</y>
                                <width>60</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>处理模式</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_execute_mode">
                        <property name="geometry">
                            <rect>
                                <x>520</x>
                                <y>10</y>
                                <width>61</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>执行</string>
                        </property>
                    </widget>
                    <widget class="QComboBox" name="manual_comboBox_select_format_mode">
                        <property name="geometry">
                            <rect>
                                <x>425</x>
                                <y>10</y>
                                <width>81</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <item>
                            <property name="text">
                                <string>无</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>换行校验</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>去除广告</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>词语纠错</string>
                            </property>
                        </item>
                    </widget>
                    <widget class="Line" name="line_2">
                        <property name="geometry">
                            <rect>
                                <x>345</x>
                                <y>10</y>
                                <width>16</width>
                                <height>151</height>
                            </rect>
                        </property>
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                    </widget>
                    <widget class="Line" name="line_3">
                        <property name="geometry">
                            <rect>
                                <x>370</x>
                                <y>45</y>
                                <width>210</width>
                                <height>16</height>
                            </rect>
                        </property>
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                    </widget>
                    <widget class="QLineEdit" name="manual_input_select_text">
                        <property name="geometry">
                            <rect>
                                <x>375</x>
                                <y>64</y>
                                <width>200</width>
                                <height>25</height>
                            </rect>
                        </property>
                    </widget>
                    <widget class="QLineEdit" name="manual_input_replace_text">
                        <property name="geometry">
                            <rect>
                                <x>375</x>
                                <y>97</y>
                                <width>200</width>
                                <height>25</height>
                            </rect>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_select_text">
                        <property name="geometry">
                            <rect>
                                <x>365</x>
                                <y>130</y>
                                <width>51</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>查询</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_replace_text">
                        <property name="geometry">
                            <rect>
                                <x>445</x>
                                <y>130</y>
                                <width>51</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>替换</string>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="manual_button_replace_text_all">
                        <property name="geometry">
                            <rect>
                                <x>515</x>
                                <y>130</y>
                                <width>61</width>
                                <height>26</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>替换全部</string>
                        </property>
                    </widget>
                </widget>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>220</y>
                        <width>60</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>信息打印</string>
                </property>
            </widget>
            <widget class="Line" name="line">
                <property name="geometry">
                    <rect>
                        <x>70</x>
                        <y>220</y>
                        <width>620</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
            </widget>
            <widget class="QPlainTextEdit" name="plainTextEdit">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>240</y>
                        <width>680</width>
                        <height>335</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>700</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
